packages/cmd/src/services/convert/vitest/instances.ts

```diff
@@ -0,0 +1,84 @@
+import { generateShortHash } from '@lib/hash';
```

Comment:
this file is identical to postman since postman and vitest are both XML based input. 
lets remove it.